package.json

```diff
@@ -19,6 +19,9 @@
     "gatsby-plugin-typescript": "^2.0.14",
     "gatsby-source-filesystem": "^2.0.37",
     "gatsby-transformer-sharp": "^2.1.19",
+    "googleapis": "^39.2.0",
+    "lint": "^0.7.0",
+    "nodemailer": "^6.3.0",
```

Comment:
Do we need the dependencies in this package.json? 

I assume it is enough to have it in the functions/package.json.

Reply:
So i tried to remove googleapis and nodemailer and then tested the function out, it for some reasons didn't send any mail. I put back these dependencies in package.json and then it worked. Any idea why?

Reply:
Ok it worked. Seems like there was a mismatch of the firebase config names